sql/src/test/java/io/crate/planner/node/dql/NestedLoopPhaseTest.java

```diff
@@ -58,7 +58,8 @@ public void testSerialization() throws Exception {
                 ImmutableList.<Projection>of(),
                 DistributionInfo.DEFAULT_BROADCAST);
         SqlExpressions sqlExpressions = new SqlExpressions(T3.SOURCES, T3.TR_1);
-        Symbol filterCondition = sqlExpressions.normalize(sqlExpressions.asSymbol("a = 'foo'"));
+        Symbol joinCondition = sqlExpressions.normalize(sqlExpressions.asSymbol("t1.x = t1.i"));
```

Comment:
this is just about testing the serialization of the symbols, interpretation of the symbols is not relevant